business/dashboards.go

```diff
@@ -243,26 +341,98 @@ func (in *DashboardsService) buildRuntimesList(namespace string, templatesNames
 		if dashboard == nil {
 			continue
 		}
-		runtime := dashboard.Spec.Runtime
-		ref := models.DashboardRef{
-			Template: dashboard.Name,
-			Title:    dashboard.Spec.Title,
-		}
-		found := false
-		for i := range runtimes {
-			rtObj := &runtimes[i]
-			if rtObj.Name == runtime {
-				rtObj.DashboardRefs = append(rtObj.DashboardRefs, ref)
-				found = true
-				break
+		runtimes = addDashboardToRuntimes(dashboard, runtimes)
+	}
+	return runtimes
+}
+
+func (in *DashboardsService) fetchMetricNames(namespace, app, version string) []string {
+	labels := fmt.Sprintf(`{namespace="%s",app="%s"`, namespace, app)
+	if version != "" {
+		labels += fmt.Sprintf(`,version="%s"`, version)
```

Comment:
yeah, same thought here, as we are not using app/version but reading from config, I wonder if makes sense to align here.
At least to easy maintain in the future and tell developer that it's same constant.